kolibri_server_setup.py

```diff
@@ -48,40 +49,45 @@ def stop_debconf_dialog(dccomm):
     dccomm.stdin.flush()
 
 
-def set_debconf_port(port):
+def set_debconf_ports(port, zip_content_port):
     """
     Sets the port kolibri uses in debconf database, to be used by future
     reconfigurations or upgrades of the kolibri-server package
     """
     dccomm = start_debconf_dialog()
     dccomm.stdin.write("SET kolibri-server/port {}\n".format(port))
+    dccomm.stdin.write("SET kolibri-server/zip_content_port {}\n".format(zip_content_port))
     dccomm.stdin.flush()
     stop_debconf_dialog(dccomm)
-    return port
 
 
 def set_port(port):
     """
     Modify Kolibri options to set the TCP port the server will listen on
     """
-    update_options_file("Deployment", "HTTP_PORT", port, KOLIBRI_HOME)
+    update_options_file("Deployment", "HTTP_PORT", port)
 
+def set_zip_content_port(port):
+    """
+    Modify Kolibri options to set the TCP port the hashi files will be served on
+    """
+    update_options_file("Deployment", "ZIP_CONTENT_PORT", port)
 
 def disable_cherrypy():
     """
     Disables internal kolibri web server.
     Kolibri will only run background tasks.
     Web must be provided by an external server, usually uwsgi + nginx
     """
-    update_options_file("Server", "CHERRYPY_START", False, KOLIBRI_HOME)
+    update_options_file("Server", "CHERRYPY_START", False)
```

Comment:
Could we just get rid of these functions, and instead just invoke kolibri with `kolibri services` in https://github.com/learningequality/kolibri-server/blob/master/debian/kolibri-server.service#L11 ? The `CHERRYPY_START` option is now deprecated, and hopefully can be removed in future.

Reply:
`CHERRYPY_START` is not added anymore